File: PIA_PD/ClientApp/src/logic/Movimiento.js
import axios from 'axios';

const ENDPOINT_PATH = 'api/movimiento'
export default {
    listaMovimientos() {
        return axios.get(ENDPOINT_PATH + '/Read');
    },
    registrarMovimiento(objeto){
        return axios.post(ENDPOINT_PATH+'/Create', {
            IdTipoMovimiento : Number(objeto.idTipoMovimiento),
            CantDolares : Number(objeto.cantDolares),
            PUnitario : Number(objeto.pUnitario),
            CostoTotal : Number(objeto.costoTotal),
            Pago : Number(objeto.pago),
            Cambio : Number(objeto.cambio),
            IdUsuario : Number(localStorage.IdUsuario)
        });
    }
}

File: Models/MOVIMIENTO.cs
namespace Models
{
    public class MOVIMIENTO
    {
        public int IdMovimiento { get; set; }
        public int IdTipoMovimiento { get; set; }
        public int CantDolares { get; set; }
        public decimal PUnitario { get; set; }
        public decimal CostoTotal { get; set; }
        public decimal Pago { get; set; }
        public decimal Cambio { get; set; }
        public DateTime FecRegistro { get; set; }
        public int IdUsuario { get; set; }
        public string NombreCompleto { get; set; }

    }
}